src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -43,6 +45,37 @@ private static IVsTextView GetActiveVsTextView()
             return vsTextView;
         }
 
+        public string[] GetCurrentClassifications()
```

Comment:
> If you have a selection, this doesn't go grab all of the classifications under the selection or anything like that.

If you have no selection it will find the classifications for the word the caret is on (`GetExtentOfWord(selectionSpan.Start).Span`)

If you do have a selection, it returns all of the classifications for that selection (`classifier.GetClassificationSpans(selectionSpan)`)

---
In reply to: [104517732](https://github.com/dotnet/roslyn/pull/17546#discussion_r104517732) [](ancestors = 104517732)